TkAlZMuMu/ZMM_13TeV_TuneCUETP8M1_cfi_GEN_SIM_DIGI_L1_DIGI2RAW_RAW2DIGI_L1Reco_RECO_ALCA_Templ.py

```diff
@@ -138,9 +154,20 @@ def customise_alcareco(process):
 )
 
 # Other statements
+process.mix.input.nbPileupEvents.averageNumber = cms.double(50.000000)
+process.mix.bunchspace = cms.int32(25)
+process.mix.minBunch = cms.int32(-12)
+process.mix.maxBunch = cms.int32(3)
+process.mix.input.fileNames = cms.untracked.vstring(['/store/relval/CMSSW_10_6_0/RelValMinBias_13/GEN-SIM/106X_upgrade2021_realistic_v4_rsb-v1/10000/951D01A7-EA63-5140-924F-39CDAC485C80.root', '/store/relval/CMSSW_10_6_0/RelValMinBias_13/GEN-SIM/106X_upgrade2021_realistic_v4_rsb-v1/10000/C62AF47C-4632-1C48-9DCF-D21A342757AB.root', '/store/relval/CMSSW_10_6_0/RelValMinBias_13/GEN-SIM/106X_upgrade2021_realistic_v4_rsb-v1/10000/86682229-A45E-BF45-9CF7-42E90E8D80A3.root', '/store/relval/CMSSW_10_6_0/RelValMinBias_13/GEN-SIM/106X_upgrade2021_realistic_v4_rsb-v1/10000/050252BF-B45B-6749-BB24-67251BE4F0FF.root', '/store/relval/CMSSW_10_6_0/RelValMinBias_13/GEN-SIM/106X_upgrade2021_realistic_v4_rsb-v1/10000/FC4B4722-0BB2-6141-9D8C-F14B0BD6928B.root', '/store/relval/CMSSW_10_6_0/RelValMinBias_13/GEN-SIM/106X_upgrade2021_realistic_v4_rsb-v1/10000/D3BF18BE-4C61-C44C-9DA8-72ACB515D2D2.root'])
```

Comment:
Hi @adewit do we actually need to simulate PU for the Z->mm? The ALCARECO producer anyway retains only muons.

Reply:
Ah right of course, indeed don't need to bother with it in that case

Reply:
can you clean that part? I will merge it then.